draft-ietf-privacypass-architecture.md

```diff
@@ -834,7 +848,7 @@ described in {{issue-flow}}, if the Issuer produces tokens for a single Origin,
 then per-Origin tokens are not appropriate since the Attester can infer the
 Origin from a token request.
 
-# Centralization Considerations
+# Centralization Considerations {#centralization}
```

Comment:
Did we mean to cite this? I don't see a citation above.

Reply:
I wanted to but then didn't, so I just left the change in for my future self.